### PR TITLE
Update payment tracking object diagrams and remove sequence diagram

### PR DESCRIPTION
**Pull request by atyhamos**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -181,53 +181,80 @@ of the second tutee found on the list. The information to be listed is based on
...
 * `isOverdue` — a boolean flag which denotes if the payment is overdue
 
-Parsing the user's input through `PaymentCommand`, the user may:
-* `Payment#addPayment(Lesson, int)`  — Adds the cost of `Lesson` to total fees incurred, `int` times
-* `Payment#editPayment(float)`  — Updates the total fees incurred to the specified `float` amount
-* `Payment#setPayByDate(LocalDate)`  — Updates the pay-by date for the Tutee to the specified `LocalDate`
-* `Payment#receivePayment()`  — Resets the Tutee's incurred fees and updates their payment history
+Parsing the user's input through `PaymentCommandParser`, the user may generate any one of the following commands:
+* `PaymentAddCommand`  — Adds the cost of the lesson's fees to the tutee's current payment amount due
+* `PaymentSetAmoutnCommand`  — Sets the payment amount due for the tutee to the specified amount
```
Comment (ykwei7) [inline docs/DeveloperGuide.md:202]: Minor typo here!

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -181,53 +181,80 @@ of the second tutee found on the list. The information to be listed is based on
...
 
 Step 1. The user adds a new `Tutee` John to Track-O and the `Payment` object is initialized with default values.
 
-![PaymentTracking1](images/PaymentTracking1.png)
+<p align="center">
+    <img alt="PaymentTracking1" src="images/PaymentTracking1.png"/>
+</p>
 
-Step 2. After adding lessons to John, the user executes "payment 1 add/2 l/1", where John is index `1` in the `Tutee` list, and `lesson1` is index `1` in the `Lesson` set.
+Step 2. After adding lessons to John, the user executes "payment 1 lesson/1", where John is index `1` in the `Tutee` list, and `lesson1` is index `1` in the `Lesson` set.
```
Comment (ykwei7) [inline docs/DeveloperGuide.md:215]:
```suggestion
Step 2. After adding lessons to John, the user executes "payment 1 lesson/1", where John is index `1` in the `Tutee` list, and `lesson1` is index `1` in the `Lesson` list.
```

Set has been updated to list

**Review by ykwei7 — APPROVED** [review]
Other than mentioned minor changes, rest looks good to me! Will take a look at the UML diagrams once it is merged in.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (atyhamos) [reply docs/DeveloperGuide.md]: Thanks for spotting that! Resolved

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (atyhamos) [reply docs/DeveloperGuide.md]: Thanks for spotting that! Resolved.

**Pull request merged**